### PR TITLE
Show node version in the terminal prompt

### PR DESCRIPTION
**Pull request by treznick**
and do the right thing if you have nvm or not

### CODE REVIEW

**Comment by jnimety** [comment]
What does the new prompt look like?

**Comment by treznick** [comment]
screen shot 2017-07-27 at 10 11 06 am

@jnimety

**Comment by jnimety** [comment]
:+1:

File: home/.oh-my-zsh-custom/continuity.zsh-theme [hunk 1]
```diff
@@ -7,8 +7,13 @@ if hash rvm-prompt 2>/dev/null; then
   local rvm_ruby='%{$fg[red]%}‹$(rvm-prompt s i v p g)›%{$reset_color%}'
 fi
 local git_branch='$(git_prompt_info)%{$reset_color%}'
+if type nvm >/dev/null 2>&1; then
+  local nvm_node='%{$fg[green]%}‹node-$(nvm current 2>/dev/null)›%{$reset_color%}'
+else
+  local nvm_node='%{$fg[green]%}‹node-$(node --version)›%{$reset_color%}'
```
Comment (treznick) [inline home/.oh-my-zsh-custom/continuity.zsh-theme]: oh wait what if you don't have `node`?

**Comment by treznick** [comment]
~let's hold off on this until I address my comment~

File: home/.oh-my-zsh-custom/continuity.zsh-theme (hunk 1 above)
Comment (benjaminoakes) [reply home/.oh-my-zsh-custom/continuity.zsh-theme]: This variable name is a little weird... it's not `nvm_node` in this case, right?

Comment (treznick) [reply home/.oh-my-zsh-custom/continuity.zsh-theme]: good point @benjaminoakes . `node_version_prompt` might be a bit more intent revealing

Comment (treznick) [reply home/.oh-my-zsh-custom/continuity.zsh-theme]: Fixed

**Review by benjaminoakes — APPROVED** [review]
_resurrects a 2 year old PR_ 😅

LGTM

Looks like all blockers are resolved.  Ready to merge or is this way out of date @treznick?

**Comment by benjaminoakes** [comment]
Just cleaning up PRs I've touched

Closing because it's been 2 1/2 years.  Would be happy to see it reopened if it's ready to merge.

**Pull request closed without merging**